package.json

```diff
@@ -48,6 +48,8 @@
     "test:ui:runner": "grunt test:ui:runner",
     "test:server": "grunt test:server",
     "test:coverage": "grunt test:coverage",
+    "test:jest": "node scripts/jest --watch",
```

Comment:
The point of moving to the scripts directory is to remove the dependency of `npm run`. I think we should remove these scripts and rely/document the actual `scripts/jest` script